torch/csrc/autograd/functions/accumulate_grad.h

```diff
@@ -140,7 +140,7 @@ struct TORCH_API AccumulateGrad : public Node {
             new_grad.sizes(),
             new_grad.options()));
       } else {
-        if (new_grad.is_sparse() || new_grad.is_sparse_csr()) {
+        if (new_grad.is_sparse() || new_grad.is_sparse_csr() || new_grad.is_nested()) {
```

Comment:
piggy backed here to just call clone

Reply:
Why do you want the `.clone()` behavior here? Why is the default one not working for you?